include/truffle.h

```diff
@@ -126,6 +127,8 @@ void truffle_write_idx_b(void *object, int idx, bool value);
 
 // Strings
 void *truffle_read_string(const char *string);
+void *truffle_read_bytes(const char *bytes);
```

Comment:
Ah naming. Maybe read_cstring, because unlike read_string it does not create a Java String?